rest/kadai-rest-spring-test-lib/src/main/resources/kadai-test.ldif

```diff
@@ -428,13 +431,24 @@ permission: Kadai:CallCenter:AB:AB/A:CallCenter
 cn: g01
 objectclass: groupofuniquenames
 objectclass: top
+pid: Kadai:CallCenter:AB:AB/A:CallCenter
 
 dn: cn=g02,cn=groups,OU=Test,O=KADAI
 uniquemember: uid=user-1-2,cn=users,OU=Test,O=KADAI
 permission: Kadai:CallCenter:AB:AB/A:CallCenter-vip
 cn: g02
 objectclass: groupofuniquenames
 objectclass: top
+pid: Kadai:CallCenter:AB:AB/A:CallCenter-vip
```

Comment:
maybe this should be different to "permission" in the tests?

Reply:
It has to be the same for these two permissions, because they are used in the tests that are run on both setups, the setup with permissionIdAttribute and the setup without the attribute. In order for these tests to pass, the values of the two attributes have to be the same. That's why there is a new permission that is not part of the other test. The new permission has different values